### PR TITLE
default static route test

### PR DESCRIPTION
**Pull request by mastarkey**
static route test with the following steps

- Test case steps:
1.  Configures DUT interfaces port1 and port2 with VRF name VRF1
2. Configure DUT Interface with IPv4 and IPv6 Address
4. Configures OTG interfaces and creates 
5. Create Traffic flow with Src address of OTG port 1 interface and dst address of 203.0.113.x
6. Configure default static route under VRF VRF1 
7. Verify traffic flow has no loss

### CODE REVIEW

**Review by nraghavendran — COMMENTED** [review]
LGTM btu will let the other reviewer also complete their review

**Review by mihirpitale-googler — APPROVED** [review]

File: feature/staticroute/tests/static_route_test.go [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+		otg.PushConfig(t, top)
+		otg.StartProtocols(t)
+
+	})
+
+	t.Run(fmt.Sprintf("Start traffic and verify traffic"), func(t *testing.T) {
+		verifyTrafficStreams(t, ate, top, otg, v4flow, v6flow)
+	})
+
+	t.Log("done")
```
Comment (nraghavendran) [inline feature/staticroute/tests/static_route_test.go:305]: pls rephrase comment

**Review by nraghavendran — DISMISSED** [review]
1 minor comment + static analysis failures need to be fixed

**Review by AmrNJ — APPROVED** [review]
LGTM.

**Comment by mastarkey** [comment]
Hi @mihirpitale-googler / @nraghavendran / @AmrNJ 

If there are no pending change request can we kindly merge this its been sitting for some time.

**Review by AmrNJ — APPROVED** [review]

**Pull request merged**